UnitsNet/GeneratedCode/Quantities/MassConcentration.g.cs

```diff
@@ -41,6 +44,11 @@ namespace UnitsNet
     [DataContract]
     public readonly partial struct MassConcentration :
         IArithmeticQuantity<MassConcentration, MassConcentrationUnit, double>,
+#if NET7_0_OR_GREATER
+        IMultiplyOperators<MassConcentration, Volume, Mass>,
+        IDivisionOperators<MassConcentration, MolarMass, Molarity>,
+        IDivisionOperators<MassConcentration, Density, VolumeConcentration>,
+#endif
```

Comment:
Nice touch on adding this 👍

Reply:
That was the main goal of this exercise, the generated operators where just the way to get there 😄